source/style-guide/information-types/prototypes/reference/server-params-style.txt

```diff
@@ -0,0 +1,129 @@
+.. _meta-reference-server-params:
+
+=======================
+Server Parameters Style
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+When you document a new server parameter, include the following details 
+in addition to specific information about the new parameter. These details
+ought to be documented in the order they appear in the following table:
+
+.. list-table::
+   :widths: 20 20 60
+   :header-rows: 1
+
+   * - Documentation Detail
+     - Necessity
+     - Description
+
+   * - ``mongod`` / ``mongos`` Support
+     - Required
+     - Whether the parameter is supported on ``mongod``, ``mongos``, or both.
+       Use the ``|mongod-only|``, ``|mongos-only|``, or ``|both|`` substitutions
+       as appropriate.
+
+   * - Version the Parameter was Added or Changed In
+     - Conditional
+     - 
+       - For recent versions, include the version the parameter was added 
+         or changed in.
+       - For archived versions, delete the version added tag.
+
+   * - OS Availability
+     - Optional
+     - Some parameters are available only on certain operating systems. If the 
+       parameter is supported only on certain operating systems, specify which 
+       it's supported on.
+
+   * - Argument Type
+     - Required
+     - Type of argument the parameter expects: integer, string, boolean,
+       document, and so on.
+
+   * - Default Value
+     - Required
+     - Specify the default value of the parameter.
+   
+   * - Startup and Runtime Support
+     - Required
+     - Specify whether the parameter is supported at startup, runtime, or both.
+       For details, see :ref:`startup-runtime-support`.
+
+.. _startup-runtime-support:
+
+Startup and Runtime Support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
[Minor]

Since this is now an h1, let's use the hyphen underline `-` rather than tilde `~`. Same comment applies to the "Example" heading.

Reply:
Done. Thanks!